### PR TITLE
first pass at n64 support

### PR DESCRIPTION
**Pull request by lifning**
it doesn't crash with parallel n64~
the video frame shows up screwy because we don't yet support the pixel format, but it's recognizable and runs at a reasonable speed.

### CODE REVIEW

File: py_retro/core.py [hunk 1]
```diff
@@ -178,6 +178,8 @@ def _reset_vars(self):
         self._game_loaded = False
         self.env_props = {}
         self.env_vars = {}
+        # HACK: just put this in for software frames 'til we support SET_HW_RENDER
```
Comment (vivlim) [inline py_retro/core.py:181]: please create a corresponding issue to track

File: py_retro/wave_audio.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def set_audio_sink_batch(core, filenameOrHandle):
     def wrapper(data, frames):
         # We can safely use .writeframesraw() here because the header will be
         # corrected once we call .close()
-        res.writeframesraw(ctypes.string_at(data, frames * 2)[:frames * 2])
+        res.writeframesraw(ctypes.string_at(data, frames * 4)[:frames * 4])
```
Comment (vivlim) [inline py_retro/wave_audio.py]: what are 2 & 4 here? please add a comment or use a named constant

**Review by vivlim — APPROVED** [review]

File: py_retro/wave_audio.py (hunk 2 above)
Comment (vivlim) [reply py_retro/wave_audio.py]: also, does this break other cores? should it be configurable?

Comment (lifning) [reply py_retro/wave_audio.py]: it's always been broken, actually.  naming the constants would be a good idea~
it's basically number of frames * sizeof(int16_t) * stereo

**Pull request merged**